### PR TITLE
Plugins/Themes: update messaging from "upload" to "install"

### LINKED ISSUE #33062: Plugins/Themes: Install or upload? Messaging should be consistent
Screen Shot 2019-05-15 at 12 15 13 PM

We use two different terms ("Upload" and "install) for the same action on the same page, and throughout the Plugins section.

* Can we make these consistent and use one or the other going forward? 
* Which term is less jargon-y and easier to understand? How does this compare to other services that offer third-party plugin integration?
* We should also consider using the same terminology for uploading/installing third-party themes.

Found during this walkthrough by @ianstewart https://cloudup.com/cTAV7Njdy1z

### PR DESCRIPTION
**Pull request by jaclync**
#### Changes proposed in this Pull Request

Updated messaging from "upload" to "install" for the following cases:
* Plugins
  - Install page title and success notice
  - Plugin browser page
* Themes
  - Plans page for theme features and FAQ
  - Theme showcase page
  - Theme install page title and error notice

#### Testing instructions

scenario | before | after
-- | -- | --
Plugin browser page | Screen Shot 2019-09-14 at 11 00 13 AM | Screen Shot 2019-09-14 at 11 07 55 AM
Plugin install page | Screen Shot 2019-09-14 at 12 24 24 PM | Screen Shot 2019-09-14 at 11 17 57 AM
Plugin install success notice | Screen Shot 2019-09-14 at 11 23 08 AM | Screen Shot 2019-09-14 at 11 28 00 AM
Theme showcase page | Screen Shot 2019-09-14 at 11 46 14 AM | Screen Shot 2019-09-14 at 11 47 31 AM
Theme install page | Screen Shot 2019-09-14 at 11 47 42 AM | Screen Shot 2019-09-14 at 11 52 55 AM
Theme install error (e.g. invalid file type) | Screen Shot 2019-09-14 at 11 48 55 AM | Screen Shot 2019-09-14 at 11 53 06 AM
Plans > theme install FAQ | Screen Shot 2019-09-14 at 11 36 11 AM | Screen Shot 2019-09-14 at 11 52 38 AM
Plans > theme feature list | Screen Shot 2019-09-14 at 11 36 15 AM | Screen Shot 2019-09-14 at 11 52 30 AM

Fixes #33062

### CODE REVIEW

File: client/blocks/upload-drop-zone/index.jsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ class UploadDropZone extends Component {
 
 	render() {
 		const { translate, disabled } = this.props;
-		const dropText = translate( 'Drop files or click here to upload' );
+		const dropText = translate( 'Drop files or click here to install' );
```
Comment (a8ci18n) [inline client/blocks/upload-drop-zone/index.jsx]: :ok: This change is safe, we'll keep the existing 22 translations.

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 2]
```diff
@@ -71,9 +71,9 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			/>
 
 			<FAQItem
-				question={ translate( 'Can I upload my own theme?' ) }
+				question={ translate( 'Can I install my own theme?' ) }
 				answer={ translate(
-					"Yes! With the WordPress.com Business plan you can upload any theme you'd like." +
+					"Yes! With the WordPress.com Business plan you can install any theme you'd like." +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Yes! With the WordPress.com Business plan you can upload any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/plugins/plugins-browser/index.jsx [hunk 3]
```diff
@@ -475,8 +475,8 @@ export class PluginsBrowser extends Component {
 		return (
 			<HeaderButton
 				icon="cloud-upload"
-				label={ translate( 'Upload Plugin' ) }
-				aria-label={ translate( 'Upload Plugin' ) }
+				label={ translate( 'Install Plugin' ) }
```
Comment (a8ci18n) [inline client/my-sites/plugins/plugins-browser/index.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Install plugin' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/install-action-messaging)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/themes/theme-showcase.jsx [hunk 4]
```diff
@@ -246,7 +246,7 @@ class ThemeShowcase extends React.Component {
 							href={ siteSlug ? `/themes/upload/${ siteSlug }` : '/themes/upload' }
 						>
 							<Gridicon icon="cloud-upload" />
-							{ translate( 'Upload Theme' ) }
+							{ translate( 'Install Theme' ) }
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-showcase.jsx]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Install Themes', 'NULL', { context: 'NULL'} )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/install-action-messaging)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/themes/theme-upload/index.jsx [hunk 5]
```diff
@@ -136,14 +136,14 @@ class Upload extends React.Component {
 		debug( 'Error', { error } );
 
 		const errorCauses = {
-			exists: translate( 'Upload problem: Theme already installed on site.' ),
-			already_installed: translate( 'Upload problem: Theme already installed on site.' ),
-			'too large': translate( 'Upload problem: Theme zip must be under 10MB.' ),
-			incompatible: translate( 'Upload problem: Incompatible theme.' ),
-			unsupported_mime_type: translate( 'Upload problem: Not a valid zip file' ),
+			exists: translate( 'Install problem: Theme already installed on site.' ),
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-upload/index.jsx]: :disappointed: 20 existing translations will be lost with this change.

File: client/my-sites/themes/theme-upload/index.jsx [hunk 6]
```diff
@@ -136,14 +136,14 @@ class Upload extends React.Component {
...
+			already_installed: translate( 'Install problem: Theme already installed on site.' ),
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-upload/index.jsx]: :ok: This change is safe, we'll keep the existing 20 translations.

File: client/my-sites/themes/theme-upload/index.jsx [hunk 7]
```diff
@@ -136,14 +136,14 @@ class Upload extends React.Component {
...
+			'too large': translate( 'Install problem: Theme zip must be under 10MB.' ),
+			incompatible: translate( 'Install problem: Incompatible theme.' ),
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-upload/index.jsx]: :ok: This change is safe, we'll keep the existing 21 translations.

File: client/my-sites/themes/theme-upload/index.jsx [hunk 8]
```diff
@@ -136,14 +136,14 @@ class Upload extends React.Component {
...
+			unsupported_mime_type: translate( 'Install problem: Not a valid zip file' ),
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-upload/index.jsx]: :ok: This change is safe, we'll keep the existing 22 translations.

File: client/my-sites/themes/theme-upload/index.jsx [hunk 9]
```diff
@@ -136,14 +136,14 @@ class Upload extends React.Component {
...
 			initiate_failure: translate(
-				'Upload problem: Theme may not be valid. Check that your zip file contains only the theme ' +
-					'you are trying to upload.'
+				'Install problem: Theme may not be valid. Check that your zip file contains only the theme ' +
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-upload/index.jsx]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Upload problem: Theme may not be valid. Check that your zip file contains only the theme you are trying to upload.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/themes/theme-upload/index.jsx [hunk 10]
```diff
@@ -267,7 +267,7 @@ class Upload extends React.Component {
 				<QueryActiveTheme siteId={ siteId } />
 				{ themeId && complete && <QueryCanonicalTheme siteId={ siteId } themeId={ themeId } /> }
 				<ThanksModal source="upload" />
-				<HeaderCake backHref={ backPath }>{ translate( 'Upload theme' ) }</HeaderCake>
+				<HeaderCake backHref={ backPath }>{ translate( 'Install theme' ) }</HeaderCake>
```
Comment (a8ci18n) [inline client/my-sites/themes/theme-upload/index.jsx]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Install Themes', 'NULL', { context: 'NULL'} )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/install-action-messaging)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by gwwar** [comment]
This has failing tests I think due to the updates to gridicon import updates, @sirreal is attempting to update usages in https://github.com/Automattic/wp-calypso/pull/36143 so let's remove these changes from this PR. I'll update the branch.

**Pull request merged**